HipoProject/urls.py

```diff
@@ -29,6 +29,8 @@
     path('api/profiles/authenticate/', UserAuthenticationView.as_view()),
     path('api/profiles/me/', MyProfileDetailView.as_view()),
     path('api/profiles/me/change-password/', ChangePasswordView.as_view()),
+    path('api/ingredients/create/', CreateIngredientView.as_view(), name='create_ingredient'),
+    path('api/ingredients/list/', ListIngredientView.as_view(), name='list_ingredient'),
```

Comment:
You can use the same endpoint to create and list. URI should be `api/ingredients/`. You can check `ListCreateAPIView`.